src/elements/buildJobElement.ts

```diff
@@ -1,30 +1,39 @@
 import { Job, JobKind } from '../jobs/types';
-import type { JobElement, ElementHash } from './types';
+import { JobElement, ElementHash, ElementKind } from './types';
+
+export const jobKindCopyMap = {
+	[JobKind.createFile]: 'Create',
+	[JobKind.deleteFile]: 'Delete',
+	[JobKind.moveAndRewriteFile]: 'Move & Rewrite',
+	[JobKind.moveFile]: 'Move',
+	[JobKind.copyFile]: 'Copy',
+	[JobKind.rewriteFile]: 'Rewrite',
+};
 
 export const buildJobElement = (job: Job, rootPath: string): JobElement => {
 	const oldPath = job.oldUri?.fsPath.replace(rootPath, '') ?? '';
 	const newPath = job.newUri?.fsPath.replace(rootPath, '') ?? '';
 
-	let label: string;
+	let label: string = jobKindCopyMap[job.kind];
```

Comment:
wasn't the previous way of creating the label easier to read?

Reply:
I created `jobKindCopyMap` to not duplicate if statements in [here](https://github.com/intuita-inc/intuita-vscode-extension/blob/folder-breakdown/src/components/webview/MainWebviewProvider.ts#L166)

Reply:
But that doesn't mean we have to use this map in this file. I can revert the change in this file.